R/rhtmlCombinedScatter.R

```diff
@@ -427,8 +427,18 @@ CombinedScatter <- function(
         }
         color.min <- min(color.tmp, na.rm = TRUE)
         color.max <- max(color.tmp, na.rm = TRUE)
+
+        # Convert color variable to scale on 0 to 1
         colors.scaled <- (color.tmp - color.min)/(color.max - color.min)
         colors <- rgb(color.func(colors.scaled), maxColorValue = 255)
+
+        # Use existing color interpolation to reduce the number
+        # colors passed to plotly otherwise it will be ignored
+        if (length(color.scale) >= 50)
```

Comment:
Is 50 the threshold? Does the bug occur at 49?

Reply:
Yep, 50 is the threshold. The bug doesn't occur at 49, but it happens once you change it to 50